msteams-platform/TOC.yml

```diff
@@ -792,6 +792,14 @@
       - name: Build Adaptive Card-based Loop component
         href: m365-apps/cards-loop-component.md
         displayName: loop component, weburl, metadata
+  - name: M365 Actions
```

Comment:
Global comment: Always spell out Microsoft 365 ("M365" isn't sanctioned by branding for public comms)

Reply:
Also I think the spacing is misaligned here, so the new articles appear orphaned rather than hooked into the TOC.